src/applications/hca/tests/00-all-fields.e2e.spec.js

```diff
@@ -224,7 +234,11 @@ module.exports = E2eHelpers.createE2eTest(client => {
     .before(Timeouts.slow);
 
   // Submit message
-  client.expect.element('.confirmation-page-title').to.be.visible;
+  if (process.env.BUILDTYPE !== ENVIRONMENTS.VAGOVPROD) {
+    client.expect.element('.confirmation-page').to.be.visible;
+  } else {
+    client.expect.element('.confirmation-page-title').to.be.visible;
+  }
```

Comment:
I don't know much about this app, but is it necessary to have the logic for `e2eTestNotInMVI`, or is the feature-flag here enough to verify that the gate is working as expected? Just thinking that post-launch you'll want to tear down any feature flags so it might be easier if it touches fewer files.

Reply:
Can you make a note in your ticket to ungate the new HCA confirmation page that this test will need to be updated?

Reply:
@ncksllvn , yeah I'm doing it a different way now, w/ Erik's help, to avoid touching the reducer code.

@erikphansen , will do.  Thx for the headsup!